2025-04-28 09:09:28,738 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-04-28 09:09:43,751 - ERROR - Request Error http://89.45.12.66:8080/live/toptv_207670/LgNXVm3I/14161.m3u8: HTTPConnectionPool(host='89.45.12.66', port=8080): Max retries exceeded with url: /live/toptv_207670/LgNXVm3I/14161.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd58710>, 'Connection to 89.45.12.66 timed out. (connect timeout=15)'))
2025-04-28 09:09:58,755 - ERROR - Request Error http://190.102.43.113/Formigatv/offilinetv2806/1541684.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/1541684.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd58ce0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-04-28 09:10:13,769 - ERROR - Request Error http://190.102.43.113/Formigatv/offilinetv2806/1541685.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/1541685.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd595b0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-04-28 09:10:13,838 - ERROR - Request Error http://15.235.15.230:14785/: HTTPConnectionPool(host='15.235.15.230', port=14785): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd59ee0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:10:13,904 - ERROR - Request Error http://15.235.15.230:14757/: HTTPConnectionPool(host='15.235.15.230', port=14757): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5a7e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:10:13,970 - ERROR - Request Error http://15.235.15.230:14517/: HTTPConnectionPool(host='15.235.15.230', port=14517): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd59dc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:10:14,037 - ERROR - Request Error http://15.235.15.230:14519/: HTTPConnectionPool(host='15.235.15.230', port=14519): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd59460>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:10:14,104 - ERROR - Request Error http://15.235.15.230:14488/: HTTPConnectionPool(host='15.235.15.230', port=14488): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd58890>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:10:14,170 - ERROR - Request Error http://15.235.15.230:14464/: HTTPConnectionPool(host='15.235.15.230', port=14464): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5aae0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:10:14,239 - ERROR - Request Error http://15.235.15.230:14462/: HTTPConnectionPool(host='15.235.15.230', port=14462): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5b440>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:10:29,255 - ERROR - Request Error http://89.45.12.66:8080/live/toptv_207670/LgNXVm3I/14146.m3u8: HTTPConnectionPool(host='89.45.12.66', port=8080): Max retries exceeded with url: /live/toptv_207670/LgNXVm3I/14146.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd5bdd0>, 'Connection to 89.45.12.66 timed out. (connect timeout=15)'))
2025-04-28 09:10:44,272 - ERROR - Request Error http://89.45.12.66:8080/live/toptv_207670/LgNXVm3I/36283.m3u8: HTTPConnectionPool(host='89.45.12.66', port=8080): Max retries exceeded with url: /live/toptv_207670/LgNXVm3I/36283.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd5b260>, 'Connection to 89.45.12.66 timed out. (connect timeout=15)'))
2025-04-28 09:10:59,288 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/1541727.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/1541727.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd583e0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-04-28 09:11:14,304 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320566.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320566.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd585f0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-04-28 09:11:29,320 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320567.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320567.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd59010>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-04-28 09:11:44,337 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320570.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320570.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd4c0b0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-04-28 09:11:59,348 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320569.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320569.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd4c9e0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-04-28 09:12:14,364 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320563.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320563.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd59040>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-04-28 09:12:29,380 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320564.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320564.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f105dd58ce0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-04-28 09:12:30,102 - WARNING - URL Error https://cdn-3.nxplay.com.br/GLOBO_SP_TK/index.m3u8: Status Code 403
2025-04-28 09:12:30,472 - INFO - URL OK: http://hls1.sua.tv:80/live/globosphd/s.m3u8
2025-04-28 09:12:30,650 - WARNING - URL Error http://xcui.vitvabc.xyz:8880/laok300/8pEZjdeMPad7/32370: Status Code 401
2025-04-28 09:12:30,820 - WARNING - URL Error http://xcui.vitvabc.xyz:8880/laok300/8pEZjdeMPad7/32266: Status Code 401
2025-04-28 09:20:19,213 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112659: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:20:45,161 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112660: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:21:01,925 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112664
2025-04-28 09:21:49,253 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112665: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:22:24,871 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112666: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:23:01,553 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112667
2025-04-28 09:23:57,638 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112668: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:24:33,715 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112669: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:25:16,826 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112670: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:26:01,546 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112671
2025-04-28 09:26:55,586 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112672: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:27:45,975 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112673: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:28:33,940 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112674: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:28:49,544 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112675: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-04-28 09:29:25,353 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112676: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:30:01,353 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112677
2025-04-28 09:30:30,610 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112678: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:31:02,327 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112679
2025-04-28 09:31:28,394 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112680: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:32:20,270 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112681: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:33:01,593 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112682
2025-04-28 09:33:17,195 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112683: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-04-28 09:33:32,796 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112684: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-04-28 09:34:01,350 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112685
2025-04-28 09:34:45,014 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112686: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:35:38,532 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112687: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:36:32,091 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112688: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:37:10,415 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112689: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:38:01,483 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112690: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:38:46,809 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112691: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:39:39,695 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112692: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:40:24,012 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112693: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:41:18,337 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112694: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-04-28 09:41:18,407 - ERROR - Request Error http://15.235.15.230:14783/: HTTPConnectionPool(host='15.235.15.230', port=14783): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd07740>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:18,476 - ERROR - Request Error http://15.235.15.230:14478/: HTTPConnectionPool(host='15.235.15.230', port=14478): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5a630>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:18,545 - ERROR - Request Error http://15.235.15.230:14492/: HTTPConnectionPool(host='15.235.15.230', port=14492): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5a000>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:18,615 - ERROR - Request Error http://15.235.15.230:14485/: HTTPConnectionPool(host='15.235.15.230', port=14485): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd58680>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:18,681 - ERROR - Request Error http://15.235.15.230:14495/: HTTPConnectionPool(host='15.235.15.230', port=14495): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd582f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:18,750 - ERROR - Request Error http://15.235.15.230:14502/: HTTPConnectionPool(host='15.235.15.230', port=14502): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd59190>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:18,819 - ERROR - Request Error http://15.235.15.230:14506/: HTTPConnectionPool(host='15.235.15.230', port=14506): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd59ac0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:18,888 - ERROR - Request Error http://15.235.15.230:14504/: HTTPConnectionPool(host='15.235.15.230', port=14504): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd598e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:18,957 - ERROR - Request Error http://15.235.15.230:14508/: HTTPConnectionPool(host='15.235.15.230', port=14508): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5a6c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:19,028 - ERROR - Request Error http://15.235.15.230:14513/: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5bf20>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:19,094 - ERROR - Request Error http://15.235.15.230:14512/: HTTPConnectionPool(host='15.235.15.230', port=14512): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5a540>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:19,161 - ERROR - Request Error http://15.235.15.230:14510/: HTTPConnectionPool(host='15.235.15.230', port=14510): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105de3be00>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:19,228 - ERROR - Request Error http://15.235.15.230:14516/: HTTPConnectionPool(host='15.235.15.230', port=14516): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105de39b20>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:19,294 - ERROR - Request Error http://15.235.15.230:14492/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14492): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5b590>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:19,361 - ERROR - Request Error http://15.235.15.230:14487/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14487): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd58230>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:19,432 - ERROR - Request Error http://15.235.15.230:14499/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14499): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5a6c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:19,498 - ERROR - Request Error http://15.235.15.230:14513/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd5af00>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:19,564 - ERROR - Request Error http://15.235.15.230:14513/: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd58ad0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:19,935 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-04-28 09:41:20,268 - WARNING - URL Error http://s1.1-cup.net:2095/solucao111hgfgh77hu78/xplus1111XXhjgyu87/51: Status Code 404
2025-04-28 09:41:20,337 - ERROR - Request Error http://15.235.15.230:14487/: HTTPConnectionPool(host='15.235.15.230', port=14487): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f105dd58200>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-04-28 09:41:21,054 - INFO - URL OK: https://cdn-3.nxplay.com.br/GLOBO_SP_TK/tracks-v1a1/mono.m3u8
2025-04-28 09:41:21,420 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-04-28 09:41:21,792 - INFO - URL OK: http://hls1.sua.tv:80/live/globosphd/s.m3u8
2025-04-28 09:41:21,979 - WARNING - URL Error http://metv.fun:80/4x6cXF/JDqRqj/90059.m3u8: Status Code 404
2025-04-28 09:41:22,163 - WARNING - URL Error http://metv.fun:80/4x6cXF/JDqRqj/91431.m3u8: Status Code 404
2025-04-28 09:41:22,382 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/90034.m3u8: Status Code 404
2025-04-28 09:41:22,560 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91421.m3u8: Status Code 404
2025-04-28 09:41:22,724 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91435.m3u8: Status Code 404
2025-04-28 09:41:22,924 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91463.m3u8: Status Code 404
2025-04-28 09:41:23,131 - WARNING - URL Error http://metv.fun:80/4x6cXF/JDqRqj/90064.m3u8: Status Code 404
2025-04-28 09:41:23,302 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/90019.m3u8: Status Code 404
2025-04-28 09:41:23,487 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/95441.m3u8: Status Code 404
2025-04-28 09:41:23,647 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91405.m3u8: Status Code 404
2025-04-28 09:41:23,812 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91413.m3u8: Status Code 404
2025-04-28 09:41:23,983 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/120082.m3u8: Status Code 404
2025-04-28 09:41:24,135 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91443.m3u8: Status Code 404
2025-04-28 09:41:24,296 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91445.m3u8: Status Code 404
2025-04-28 09:41:24,453 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/95483.m3u8: Status Code 404
2025-04-28 09:41:24,849 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/97396.m3u8: Status Code 404
2025-04-28 09:41:25,024 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91456.m3u8: Status Code 404
2025-04-28 09:41:25,227 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91460.m3u8: Status Code 404
2025-04-28 09:41:25,390 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/95448.m3u8: Status Code 404
2025-04-28 09:41:25,576 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91461.m3u8: Status Code 404
2025-04-28 09:41:25,767 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91467.m3u8: Status Code 404
2025-04-28 09:41:25,946 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91470.m3u8: Status Code 404
2025-04-28 09:41:26,108 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91471.m3u8: Status Code 404
